### PR TITLE
Persisting the selected disaster across page loads

### PR DESCRIPTION
**Pull request by RamV13**

### CODE REVIEW

File: docs/resources.js [hunk 1]
```diff
@@ -12,14 +12,31 @@ function getResources() {
...
+  // default.
   const params = new URLSearchParams(window.location.search);
-  const disaster_param = params.has('disaster') ? params.get('disaster') : disaster;
-  return inProduction() ? disaster_param : disaster;
+  if (params.has('disaster')) {
+    final_disaster = params.get('disaster');
+  } else if (localStorage.getItem('disaster')) {
+    final_disaster = localStorage.getItem('disaster');
+  } else if (!inProduction()) {
+    // Start off with Harvey for test cases since our tests assert against
```
Comment (janakdr) [inline docs/resources.js]: I think we can get rid of this by having the tests just use the Harvey url? Or you could support a cookie, although I see that localStorage is nicer.

Comment (RamV13) [reply docs/resources.js]: I just set localStorage in Cypress instead.

File: cypress/integration/integration_tests/nav_test.js [hunk 2]
```diff
@@ -6,6 +6,11 @@ describe('Integration test for the navbar', () => {
 
     // The page should now be switched to the Michael disaster.
```
Comment (janakdr) [inline cypress/integration/integration_tests/nav_test.js]: Do we want the URL-changing support at all, then?

Comment (RamV13) [reply cypress/integration/integration_tests/nav_test.js]: Removed

**Pull request merged**